config/locales/en.yml

```diff
@@ -509,7 +509,7 @@ en:
     delete: 'Delete'
     dismiss: 'Dismiss'
     find: 'Find'
-    save_and_continue: 'Save and continue'
```

Comment:
So, on this one, the key and value names do not need to match.  If you just change line 512 to be:
 
```
save_and_continue: 'Continue'
```

Then this becomes a one-line PR.